documentation/src/main/asciidoc/topics/json/server_command_credentials.json

```
@@ -0,0 +1,5 @@
+{
+  "command-credential": {
+    "command": "${build.directory}/test-classes/bin/credential.sh"
```

use the same command as in xml & yaml examples.
```suggestion
    "command": "/path/to/executable.sh arg1 arg2"
```

---

Fixed